code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -16,8 +16,9 @@
 /obj/machinery/atmospherics/portable/scrubber/examine(mob/user)
 	. = ..()
 	. += "<span class='notice'>Filters the air, placing harmful gases into the internal gas container. The container can be emptied by \
-			connecting it to a connector port. The pump can pump the air in (sucking) or out (blowing), at a specific target pressure. The powercell inside can be \
-			replaced by using a screwdriver, and then adding a new cell. A tank of gas can also be attached to the scrubber.</span>"
+			connecting it to a connector port, you're unable to have [src] both connected, and on at the same time. \
```

Comment:
Same with the [src] here, I feel like using "it" would be better.

Reply:
They seem both fine to me, `[src]` just adds extra that it's about the scrubber itself and not something else

Reply:
Yeah, definitely a matter of taste there.

Reply:
I've changed some of the `[src]` around to 'it', important feeling ones remained `[src]`.